github/github-workflow.xml

```diff
@@ -0,0 +1,77 @@
+<Workflow name="Qmasters GitHub workflow for QRadar" version="1.0" xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/Workflow/V2">
+    <Parameters>
+        <Parameter name="orgName" label="Organization Name" required="false" />
+        <Parameter name="apiToken" label="API Token" required="true" secret="true" />
+        <Parameter name="phrase" label="Search Phrase" required="false"/>
+        <Parameter name="perPage" label="Per Page" required="false" default="30" />
```

Comment:
add def for 'perPage' to 100 it is not big and why not? also in parameters file.

Reply:
changed to 100